bundles/pcmjava/tools.vitruv.applications.pcmjava.pojotransformations.java2pcm/src/tools/vitruv/applications/pcmjava/pojotransformations/java2pcm/TuidUpdatePreprocessor.xtend

```diff
@@ -0,0 +1,33 @@
+package tools.vitruv.applications.pcmjava.pojotransformations.java2pcm
+
+import tools.vitruv.framework.change.processing.impl.AbstractEChangePropagationSpecification
+import tools.vitruv.domains.java.JavaDomainProvider
+import tools.vitruv.domains.pcm.PcmDomainProvider
+import tools.vitruv.framework.correspondence.CorrespondenceModel
+import tools.vitruv.framework.change.echange.EChange
+import tools.vitruv.domains.java.echange.feature.JavaFeatureEChange
+import tools.vitruv.framework.tuid.TuidManager
+import org.eclipse.emf.ecore.EObject
+import tools.vitruv.framework.util.command.ResourceAccess
+
+class TuidUpdatePreprocessor extends AbstractEChangePropagationSpecification {
```

Comment:
This is a copy of the existing TuidUpdatePreprocessor. Should be marked as such.